src/containers/BooksForm.js

```diff
@@ -0,0 +1,22 @@
+const BooksForm = () => {
+  // eslint-disable-next-line no-unused-vars
+  const category = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  return (
+    <div>
```

Comment:
**[OPTIONAL]**
The use of fragment in this instance instead of a `div` is more efficient. There is no styling dependent on the `div` hence we can use a fragment so that no extra no would be created. We can change `<div></div>` to `<></>`